### PR TITLE
Running a simple SIR model to check equivalence to Katriel's paper

### PR DESCRIPTION
**Pull request by meirhalachmi**

### CODE REVIEW

File: src/corona_hakab_model/agent.py [hunk 1]
```diff
@@ -70,27 +73,29 @@ class AgentSnapshot:
 
 
 class InitialSickAgents:
-    EXPORT_OUTPUT_DIR = "../../output/"
+    EXPORT_OUTPUT_DIR = os.path.join(os.path.split(os.path.dirname(os.path.realpath(__file__)))[0], "output")
```
Comment (bgmoshe) [inline src/corona_hakab_model/agent.py:76]: Why not use the constant in project_structure?

File: src/corona_hakab_model/katriel_validation.py [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+        )
+        recovered = ImmuneTerminalState("Recovered", detectable=False, test_willingness=0)
+
+        ret = MedicalStateMachine(susceptible, sick)
+        sick.add_transfer(recovered, dist(self.illness_duration), ...)
+        return ret
+
+
+def run_sim():
+    population_size = 3000
```
Comment (bgmoshe) [inline src/corona_hakab_model/katriel_validation.py:93]: Shouldn't this be part of config inputed to run_simulation?

File: src/corona_hakab_model/katriel_validation.py [hunk 3]
```diff
@@ -0,0 +1,196 @@
+import matplotlib.pyplot as plt
```
Comment (bgmoshe) [inline src/corona_hakab_model/katriel_validation.py:1]: Please put this file in a folder with valid name (corona_hakab_model is stuffed with more files than my middle drawer)

File: src/corona_hakab_model/agent.py (hunk 1 above)
Comment (meirhalachmi) [reply src/corona_hakab_model/agent.py:76]: It wasn't there when I started working on it :laughing: 
where is it?

Comment (meirhalachmi) [reply src/corona_hakab_model/agent.py:76]: It wasn't there when I started working on it :laughing: 
where is it?

File: src/corona_hakab_model/katriel_validation.py (hunk 2 above)
Comment (meirhalachmi) [reply src/corona_hakab_model/katriel_validation.py:93]: Nope.
It takes too much time for a bigger population size & it doesn't really work on a smaller population size

File: src/corona_hakab_model/katriel_validation.py (hunk 3 above)
Comment (meirhalachmi) [reply src/corona_hakab_model/katriel_validation.py:1]: I did it in the beginning, but when you run it from another folder all of the relatives CSV export paths break :(
Any idea on how to fix it?

File: src/corona_hakab_model/katriel_validation.py (hunk 2 above)
Comment (bgmoshe) [reply src/corona_hakab_model/katriel_validation.py:93]: Can you please document this?

File: src/corona_hakab_model/katriel_validation.py (hunk 3 above)
Comment (bgmoshe) [reply src/corona_hakab_model/katriel_validation.py:1]: project_structure.py?

File: src/corona_hakab_model/agent.py (hunk 1 above)
Comment (bgmoshe) [reply src/corona_hakab_model/agent.py:76]: You might need to puul from develop? I hope it is in develop and not in dev4.0

**Pull request closed without merging**